### DISCUSSION TITLE
Change MP4 Record Codec H.264/H.265 ?

### DISCUSSION THREAD

**Discussion by Dotatay** [roleoroleo/yi-hack-Allwinner-v2#969] [General]
Hello,

I came across your post where you explain how to switch the high-stream codec to H.264 or H.265:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/484#issuecomment-1235411673

Is it also possible to change the codec for the recorded MP4 videos stored on the SD card?
I have a Yi Pro 2K Home (y623) and would like to save the videos with H.265 codec.

**Comment by roleoroleo** [comment]
I never tried but I think that if you change the codec for the stream, it also changes in the files.

**Reply by Dotatay** [reply]
I have now tested H.265 on the y623 and it reduces the mp4 file size by 30%.
Thank you for your support :-)
